src/api/posts/src/routes/feeds.js

```diff
@@ -34,6 +35,18 @@ feeds.get('/', async (req, res, next) => {
   );
 });
 
+//This should return number of jobs in the queue, active, failed, delayed, waiting jobs
+feeds.get('/info', async (req, res, next) => {
+  let jobNum;
+  try {
+    jobNum = await queue.count();
+  } catch (err) {
+    logger.error({ error }, "Unable to get queue's info");
+    next(error);
+  }
+  res.json({ jobNum: jobNum });
```

Comment:
Move this to after line 42, inside the try/catch.  Also, you don't need to repeat `jobNum`:

```js
res.json({ jobNum });
```